transactionparser/src/channelchanges.js

```diff
@@ -41,14 +41,14 @@ function summarizePaymentChannel(node) {
 
         // The change in the number of XRP drops allocated to this channel.
         // This is positive if this is a PaymentChannelFund transaction.
-        channel_amount_change_drops: prev.Amount ? 
+        channelAmountChangeDrops: prev.Amount ? 
           new BigNumber(final.Amount)
           .minus(new BigNumber(prev.Amount || 0))
           .toString(10) :
           undefined,
 
         // The change in the number of XRP drops already paid out by the channel.
-        channel_balance_change_drops: final.Balance ? 
+        channelBalanceChangeDrops: final.Balance ? 
```

Comment:
Is this supposed to check `prev.Balance` ?

Reply:
@sublimator I think it's the difference between `final.Balance` and `prev.Balance` (see line 53).